website/src/app/blog/may-2024-update/readme.mdx

```diff
@@ -1,270 +1,92 @@
-
-Need more throughput? Simple: deploy more Gateways. The control plane will
-automatically distribute the load across all of them.
-
-We think high availability is such a core feature in a remote access solution
-that we made failover and load balancing available **on all plans**, including
-the Starter tier. [Read more](/kb/deploy/gateways) about how it works in our
-documentation.
-
-#### Firewall hole-punching
-
-You know what's not fun? Configuring firewalls.
+<Image
+  src="/images/blog/may-2024-update/traffic-restrictions.png"
+  alt="Firezone traffic restrictions"
+  width={600}
+  height={600}
+  className="mx-auto"
+/>
 
-More precisely, configuring your organization's cloud or corporate firewalls to
-allow incoming connections from the internet. Not only is it a pain to manage at
-scale, it also exposes your organization to all kinds of security risks.
+Simply add the desired filters when creating or editing a Resource, and only
+that type of traffic will be allowed. This is useful for allowing access to some
+services running on a host but not all of them.
 
-So we rearchitected Firezone to include the same NAT traversal techniques that
-WebRTC applications have enjoyed for years now:
-[STUN](https://www.rfc-editor.org/rfc/rfc8489.html) and
-[TURN](https://www.rfc-editor.org/rfc/rfc8553), known collectively as
-[ICE](https://datatracker.ietf.org/doc/html/rfc8445).
+#### How it works
 
-As you can probably surmise from the above links, these are well-established
-standards for doing reliable NAT traversal. These have been battle-tested in the
-field for years across all kinds of products -- Firezone is only the latest to
-benefit from them.
+When you add traffic restrictions to a Resource, all Gateways that can serve
+that Resource will immediately receive the updated Resource configuration.
 
-What does this mean for you? It means you can deploy Firezone without touching a
-single firewall configuration and still enjoy the same level of performance as
-if you did. Attack surface is minimized and connections are direct. It's a
-win-win.
+Restrictions are enforced at the **Gateway** level. If a user tries to access a
+service on the host not allowed by the restriction, the Gateway will simply drop
+the traffic, preventing it from ever reaching the service in question.
 
-For the curious readers, you can find our implementation of ICE, aptly named
-"snownet", in our repository
-[here](https://github.com/firezone/firezone/tree/main/rust/connlib/snownet).
+One popular use case for traffic restrictions is segmenting access to individual
+services on a host. To do this, simply create a Resource for each service on the
+host you want to allow access to, and add the appropriate traffic restrictions
+to each one. Create an Resource with the `TCP/22` restriction to allow SSH
+access for your DevOps team, then add another Resource with the `TCP/443`
+restriction to allow access to an HTTPS service for the rest of your
+organization.
```

Comment:
Given the proposed solution we're talking about in #4939, do you think we should leave this example out so that we don't guide people towards creating multiple resources with the same address while we're working through the solution?

Reply:
yeah updated